packages/shared_preferences/shared_preferences_android/example/integration_test/shared_preferences_test.dart

```diff
@@ -757,6 +787,69 @@ void main() {
         expect(await preferences.getDouble(doubleKey, options), testDouble);
         expect(await preferences.getStringList(listKey, options), testList);
       });
+
+      testWidgets(
+          'list encoding updates to new encoding process without data loss with $backend',
```

Comment:
https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-newold-modifiers-in-code

`list encoding updates to json encoding ...`